src/core/Akka/Configuration/Pigeon.conf

```diff
@@ -522,9 +522,28 @@ akka {
     }
 
 	# extra settings that can be custom to a serializer implementation
-	serialization-settings {
-
-	}
+  serialization-settings {
+      json {
+        
+        # Used to set whether to use stringbuilders from a pool
+        # In memory constrained conditions (i.e. IOT)
+        # You may wish to turn this off
+        use-pooled-string-builder = true
+        
+        # The starting size of stringbuilders created in pool
+        # if use-pooled-string-builder is true.
+        # You may wish to adjust this number,
+        # For example if you are confident your messages are smaller or larger
+        pooled-string-builder-minsize = 2048b
+        
+        # The maximum retained size of a pooled stringbuilder
+        # if use-pooled-string-builder is true.
+        # You may wish to turn this number up if your messages are larger
+        # But do keep in mind that around 85K you'll wind up
+        # on the Large Object Heap (which may not be a bad thing...)
+        pooled-string-builder-maxsize = 32768b
+      }
```

Comment:
I tried to make sure to document the settings here. I would love some feedback on the defaults, I'm not sure if maxsize is too low or not. I think MinSize is fair here; I tried 1024B and the benchmarks came out about the same.